auth/views.py

```diff
@@ -112,5 +113,22 @@ def braintree_parse(request):
     return HttpResponse(status=204)
 
 
-def reference(request):
-    raise NotImplementedError
+def reference(request, reference_name):
+    """
+    Pass through the request to the reference implementation.
+    We have to:
+    * get the provider from the URL
+    * sign the request with OAuth
+    """
+    if reference_name not in settings.ZIPPY_CONFIGURATION:
+        raise ValueError('Unknown provider: {}'.format(reference_name))
```

Comment:
do we still have to use `ZIPPY_CONFIGURATION` for all providers? I think you don't need this key lookup. You can probably change the dict to:
````
ZIPPY_CONFIGURATION = {
}
````